pom.xml

```diff
@@ -126,25 +126,6 @@ under the License.
     </dependencies>
   </dependencyManagement>
 
-  <build>
-    <pluginManagement>
-      <!-- use the same plugin version in project and in IT tests -->
-      <!-- additionally versions upgrade will be tracked -->
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-javadoc-plugin</artifactId>
-          <version>${javadocPluginVersion}</version>
-        </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-site-plugin</artifactId>
-          <version>${sitePluginVersion}</version>
```

Comment:
The same consider use version from parent.

Reply:
The section is gone anyway so that isn't a problem anymore...I guess

Reply:
properties still is used:
https://github.com/search?q=repo%3Aapache%2Fmaven-jxr%20sitePluginVersion&type=code

Reply:
Site version in ITs is on purpose because it will might differ from what parent sets, especially with Doxia 2.0.0 coming. For the time being that is acceptable. Later, it can be removed.